CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## [4.1.7]
+
+- Fix state changes in useEffect
```

Comment:
these notes are for users so rather than the internal change description we better show the user-facing change. For example, here I believe it would be "Fix crash when creating alerts" (or something like that)